### PR TITLE
Serviceclient

### PR DESCRIPTION
**Pull request by emmuli**
Added serviceclient.js where HTTP requests have been added. Axios installed

### CODE REVIEW

File: front/src/Serviceclient.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    let newUser = {firstName: userData.firstName, 
+        lastName: userData.lastName, 
+        email: userData.email, 
+        role: userData.role,
+        phoneNumber: userData.phoneNumber,
+        description: userData.description
+    }
+    Axios.post(API + "Users/", newUser).then(response => {
+        console.dir(response)
+    });
```
Comment (essile) [inline front/src/Serviceclient.js]: Let's call the callback function here

File: front/src/Serviceclient.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+export function GetConsultantInfo(userId, callback) {
+    Axios.get(API + "Users/" + userId).then(response => {
+        callback(response.data)
+    });
+}
+
+export function DeleteUser(userId) {
+    Axios.delete(API + "Users/" + userId).then(response => {
+        console.dir(response)
+    });
```
Comment (essile) [inline front/src/Serviceclient.js]: callback

File: front/src/Serviceclient.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import Axios from "axios";
```
Comment (MikaelStenstrand) [inline front/src/Serviceclient.js]: filename is Serviceclient.js, it could be ServiceClient.js

File: front/src/Serviceclient.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+const API = "http://localhost:5000/api/";
+
+export function CreateUser(userData, callback) {
+    let newUser = {firstName: userData.firstName,
```
Comment (MikaelStenstrand) [inline front/src/Serviceclient.js]: you could add a new right after the {
i.e.

let newUser = {
firstName....

File: front/src/Serviceclient.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    });
+}
+
+export function EditProfile(userId, user) {
+    Axios.put(API + "Users/" + userId, user).then(response => {
+        console.dir(response)
+    });
+}
+
+export function GetOpportunities(callback) {
```
Comment (MikaelStenstrand) [inline front/src/Serviceclient.js]: We can remove this for now and add it when we know the API

**Pull request merged**